app/services/parser-service.js

```diff
@@ -106,6 +178,34 @@ function parseAsda (packingListJson) {
   return combineParser(establishmentNumber, packingListContents, true)
 }
 
+function parseTescoModel1 (packingListJson) {
+  const establishmentNumber = packingListJson[4].AT
+  const packingListContents = packingListJson.slice(5).map(col => ({
+    description: col.G,
+    nature_of_products: null,
+    type_of_treatment: col.AS,
+    commodity_code: null,
```

Comment:
just looking at the excel and I think the commodity code is the tariff code on column L